sdk/lusid/utilities/api_client_builder.py

```diff
@@ -85,6 +87,14 @@ def build(cls, api_secrets_filename=None, id_provider_response_handler=None, api
         config.access_token = api_token
         config.host = configuration.api_url
 
+        if api_configuration.flag_to_use_decimal is not None:
+            config.flag_to_use_decimal = api_configuration.flag_to_use_decimal
+
+        if configuration.flag_to_use_decimal is not None:
+            config.flag_to_use_decimal = configuration.flag_to_use_decimal
```

Comment:
See above, better to pass into the constructor of the config to be consistent with TCP Keep Alive setting which is a very similar type of setting.